src/store/actions/hackathonActions.js

```diff
@@ -0,0 +1,19 @@
+import * as ActionType from "./index";
+import axios from "axios";
+const contentfulUrl = "https://cdn.contentful.com";
```

Comment:
I would also be driving this sort of thing from an environment variable. Makes it easier to mock things, or if contentful provided a different url to use for testing.

Basically, we shouldn't have any hardcoded configuration in the source code (even if its unlikely to change).